package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "w3c-keys",
-  "version": "0.3.6",
+  "version": "1.0.0",
```

Comment:
the `version` in `package.json` is `1.0.0` so that's why this was modified after an `npm install`